assopy/admin.py

```diff
@@ -723,16 +723,11 @@ def do_csv_invoices(self, request, queryset):
                 'Deposit Invoice', 'SIM Invoice', 'Voucher Invoice',
                 'Billing notes')
 
-        def e(d):
-            for k, v in d.items():
-                d[k] = v.encode('utf-8')
-            return d
-
         ofile = StringIO()
         writer = csv.DictWriter(ofile, fieldnames=columns)
-        writer.writerow(dict(zip(columns, columns)))
+        writer.writerow(dict(list(zip(columns, columns))))
```

Comment:
is that doing the same thing as `writer.wrieheader()`?

Reply:
yes